client/getattr_freebsd.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"syscall"
+
+	"github.com/hugelgupf/p9/p9"
+)
+
+// GetAttr implements p9.File.GetAttr.
+//
+// Not fully implemented.
```

Comment:
Is it easy to articulate what's missing?

Reply:
I think, looking at it, that it's actually complete now.